easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.0.0-GCCcore-12.2.0.eb

```diff
@@ -30,7 +30,7 @@ dependencies = [
     ('GLib', '2.75.0'),
     ('cairo', '1.17.4'),
     ('LibTIFF', '4.4.0'),
-    ('GTK3', '3.24.35'),
+    ('GTK2', '2.24.33'),
```

Comment:
`Ghostscript` prefers `GTK3` to `GTK2`, so I would suggest removing this change from the PR

```
2315 if test "x$enable_gtk" != "xno"; then                                                                                   
2316     # Try GTK+ 3.x first...                                                                                             
2317     if test "x$PKGCONFIG" != x; then                                                                                    
2318         AC_MSG_CHECKING(for GTK+ 3.x)                                                                                   
2319         if $PKGCONFIG --exists gtk+-3.0; then                                                                           
2320             SOC_LOADER="dxmain"                                                                                         
2321             SOC_CFLAGS="`$PKGCONFIG gtk+-3.0 --cflags`"                                                                 
2322             SOC_LIBS="`$PKGCONFIG gtk+-3.0 --libs`"                                                                     
2323             AC_MSG_RESULT([yes])                                                                                        
2324         else                                                                                                            
2325             AC_MSG_RESULT([no])                                                                                         
2326         fi                                                                                                              
2327                                                                                                                         
2328         # Or resort to GTK+ 2.x                                                                                         
2329         if test "x$SOC_LOADER" = "x"; then                                                                              
2330             AC_MSG_CHECKING(for GTK+ 2.x)                                                                               
2331             if $PKGCONFIG --exists gtk+-2.0; then                                                                       
2332                 SOC_LOADER="dxmain"                                                                                     
2333                 SOC_CFLAGS="`$PKGCONFIG gtk+-2.0 --cflags`"                                                             
2334                 SOC_LIBS="`$PKGCONFIG gtk+-2.0 --libs`"                                                                 
2335                 AC_MSG_RESULT([yes])                                                                                    
2336             else                                                                                                        
2337                 AC_MSG_RESULT([no])                                                                                     
2338             fi                                                                                                          
2339         fi                                                                                                              
2340     fi                                                                                                                  
2341 fi                     
```